src/config/graphical-session/xorg.md

```diff
@@ -41,17 +41,17 @@ Modesetting requires the `mesa-dri` package, and no additional vendor-specific
 driver package.
 
 Xorg defaults to DDX drivers if they are present, so in this case modesetting
-must be explicitly selected: see [Forcing the modesetting
-driver](#forcing-the-modesetting-driver).
+must be explicitly selected: see
+[Forcing the modesetting driver](#forcing-the-modesetting-driver).
 
 For advanced configuration, see
 [modesetting(4)](https://man.voidlinux.org/modesetting.4).
 
 ### Proprietary Drivers
 
 Void also provides [proprietary NVIDIA drivers](./graphics-drivers/nvidia.md),
-which are available in the [nonfree
-repository](../../xbps/repositories/index.md#nonfree).
+which are available in the
+[nonfree repository](../../xbps/repositories/index.md#nonfree).
```

Comment:
These changes don't seem to be related to pipewire, is it a mistake?

Reply:
auto-formatting of my text editor :/ i'll revert this